src/app/common/alert-list/alert-list.component.html

```diff
@@ -0,0 +1,25 @@
+
+<div class="alert-top">
+  {{alertService.alerts}} Test
+    <alert *ngFor="let alert of alertService.alerts" [type]="alert.type" [close]="alert.close()">
```

Comment:
@Chanputhi I think the migration may not be fully satisfy, because the alerts is from the rootScope in AngularJS and it is not accessible from Angular, it may raise error, please double check that. Thank you.

Please change this pull request to draft.